tasks/2.txt

```diff
@@ -18,10 +23,5 @@ db.getCollection('order-details').aggregate(
             "Order Total Price": "$Order Total Price",
             "Total Order Discount, %": "$TotalOrderDiscount"
         }
-    },{
-        $sort: {
-            "Order Id": -1
-        }
     }]
 )
```

Comment:
Order after $group is related to grouping logic. So first sort it is good idea but we still need additional sort after $group